integration/common/openlineage/common/provider/dbt/local.py

```diff
@@ -84,6 +85,8 @@ def __init__(
         profile_name: Optional[str] = None,
         target: Optional[str] = None,
         target_path: Optional[str] = None,
+        models: Optional[Sequence[str]] = None,
```

Comment:
I think it should not be limited to local processor only. I'm not entirely sure if and how this can be done with dbt cloud but additional logic seems not to interfere with base processor class.